edge-util/src/Microsoft.Azure.Devices.Edge.Util/metrics/IMetricsCounter.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util.Metrics
+{
+    using System.Collections.Generic;
+
+    public interface IMetricsCounter
+    {
+        void Increment(long amount);
+        void Decrement(long amount);
+        void Increment(long amount, Dictionary<string, string> tags);
```

Comment:
what kind of tags will be used here? should it be modeled more specific?

Reply:
Emm, I don't think so. This is intended to be a generic Metrics API for the Edge.